Resolve new build warnings that aren't excluded in Debug

Some warnings have slipped into the project, we should resolve them to clear the output and make it easier to identify issues.

https://app.circleci.com/pipelines/github/ArangoDB-Community/arangodb-net-standard/740/workflows/83a49cc2-a112-4c45-b78f-e9650daf1135/jobs/1963
```
IndexApi/Models/IndexResponseBase.cs(105,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://www.arangodb.com/docs/stable/http/indexes-geo.html' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/Analyzer.cs(13,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://www.arangodb.com/docs/stable/analyzers.html#analyzer-names' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
TransactionApi/Models/PostTransactionResponse.cs(19,15): warning CS1570: XML comment has badly formed XML -- 'End tag 'remarks' does not match the start tag 'see'.' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
IndexApi/Models/IndexResponseBase.cs(111,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://www.arangodb.com/docs/stable/http/indexes-geo.html' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
IndexApi/Models/IndexResponseBase.cs(117,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://www.arangodb.com/docs/stable/http/indexes-geo.html' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
TransactionApi/Models/PostTransactionResponse.cs(20,1): warning CS1570: XML comment has badly formed XML -- 'Expected an end tag for element 'remarks'.' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/Analyzer.cs(13,75): warning CS1658: Unexpected character '#'. See also error CS1056. [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/Analyzer.cs(20,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://www.arangodb.com/docs/stable/analyzers.html#analyzer-types' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/Analyzer.cs(20,75): warning CS1658: Unexpected character '#'. See also error CS1056. [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/AnalyzerProperties.cs(7,20): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://www.arangodb.com/docs/stable/analyzers.html#analyzer-properties' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/AnalyzerProperties.cs(7,71): warning CS1658: Unexpected character '#'. See also error CS1056. [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/AnalyzerProperties.cs(55,24): warning CS1584: XML comment has syntactically incorrect cref attribute 'https://www.arangodb.com/docs/stable/analyzers.html#stemming' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/AnalyzerProperties.cs(55,75): warning CS1658: Unexpected character '#'. See also error CS1056. [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
AnalyzerApi/Models/DeleteAnalyzerResponse.cs(8,23): warning CS1591: Missing XML comment for publicly visible type or member 'DeleteAnalyzerResponse.Name' [/root/arangodb-net-standard/arangodb-net-standard/ArangoDBNetStandard.csproj]
```

--- a/arangodb-net-standard/AnalyzerApi/Models/Analyzer.cs
+++ b/arangodb-net-standard/AnalyzerApi/Models/Analyzer.cs
@@ -10,14 +10,14 @@ namespace ArangoDBNetStandard.AnalyzerApi.Models
         /// <summary>
         /// Name of the analyzer
         /// For rules regarding analyzer names, see
-        /// <see cref="https://www.arangodb.com/docs/stable/analyzers.html#analyzer-names"/>
+        /// https://www.arangodb.com/docs/stable/analyzers.html#analyzer-names
         /// </summary>
         public string Name { get; set; }
 
         /// <summary>
         /// Type of the analyzer
         /// For valid analyzer types, see
-        /// <see cref="https://www.arangodb.com/docs/stable/analyzers.html#analyzer-types"/>
+        /// https://www.arangodb.com/docs/stable/analyzers.html#analyzer-types
         /// </summary>
         public string Type { get; set; }
 

--- a/arangodb-net-standard/AnalyzerApi/Models/AnalyzerProperties.cs
+++ b/arangodb-net-standard/AnalyzerApi/Models/AnalyzerProperties.cs
@@ -3,8 +3,8 @@
 namespace ArangoDBNetStandard.AnalyzerApi.Models
 {
     /// <summary>
-    /// Properties of an Analyzer
-    /// <see cref="https://www.arangodb.com/docs/stable/analyzers.html#analyzer-properties"/>
+    /// Properties of an Analyzer.
+    /// See https://www.arangodb.com/docs/stable/analyzers.html#analyzer-properties
     /// </summary>
     public class AnalyzerProperties
     {
@@ -52,7 +52,7 @@ namespace ArangoDBNetStandard.AnalyzerApi.Models
         /// treated as a single token, for supported languages.
         /// Stemming support is provided by Snowball,
         /// which supports the languages listed at:
-        /// <see cref="https://www.arangodb.com/docs/stable/analyzers.html#stemming"/>
+        /// https://www.arangodb.com/docs/stable/analyzers.html#stemming
         /// </summary>
         public bool Stemming { get; set; }
     }

--- a/arangodb-net-standard/IndexApi/Models/IndexResponseBase.cs
+++ b/arangodb-net-standard/IndexApi/Models/IndexResponseBase.cs
@@ -1,6 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using System.Collections.Generic;
 
 namespace ArangoDBNetStandard.IndexApi.Models
 {
@@ -102,19 +100,19 @@ namespace ArangoDBNetStandard.IndexApi.Models
 
         /// <summary>
         /// For more information,
-        /// <see cref="https://www.arangodb.com/docs/stable/http/indexes-geo.html"/>
+        /// see https://www.arangodb.com/docs/stable/http/indexes-geo.html
         /// </summary>
         public int? MaxNumCoverCells { get; set; }
 
         /// <summary>
         /// For more information,
-        /// <see cref="https://www.arangodb.com/docs/stable/http/indexes-geo.html"/>
+        /// see https://www.arangodb.com/docs/stable/http/indexes-geo.html
         /// </summary>
         public int? BestIndexedLevel { get; set; }
 
         /// <summary>
         /// For more information,
-        /// <see cref="https://www.arangodb.com/docs/stable/http/indexes-geo.html"/>
+        /// see https://www.arangodb.com/docs/stable/http/indexes-geo.html
         /// </summary>
         public int? WorstIndexedLevel { get; set; }
     }

--- a/arangodb-net-standard/TransactionApi/Models/PostTransactionResponse.cs
+++ b/arangodb-net-standard/TransactionApi/Models/PostTransactionResponse.cs
@@ -13,7 +13,7 @@ namespace ArangoDBNetStandard.TransactionApi.Models
         /// </summary>
         /// <remarks>
         /// Note that in cases where an error occurs, the ArangoDBNetStandard
-        /// client will throw an <see cref="ApiErrorException"> rather than
+        /// client will throw an <see cref="ApiErrorException"/> rather than
         /// populating this property. A try/catch block should be used instead
         /// for any required error handling.
         /// </remarks>

